webapp/src/main/java/life/qbic/usermanagement/registration/EmailConfirmationLinkSupplier.java

```diff
@@ -11,15 +11,24 @@
 @Service
 public class EmailConfirmationLinkSupplier {
 
-  @Value("${server.address}")
-  private String host;
-  @Value("${server.port}")
-  private String port;
-  @Value("${email-confirmation-endpoint}")
-  private String emailConfirmationRoute;
+
+  private final String host;
+
+  private final String port;
+
+  private final String emailConfirmationEndpoint;
+
+  public EmailConfirmationLinkSupplier(@Value("${server.address}") String host,
```

Comment:
readability was better before but this is NIT

Reply:
Yes, but it is better to inject dependencies using the constructor than using reflection.